Example/Tests/Tests.swift

```diff
@@ -41,6 +41,33 @@ class Tests: XCTestCase {
         XCTAssertEqual(issuer, "https://example.com/oauth2/authServerId")
     }
 
+    func testAdditionParamParse() {
```

Comment:
Any reason you prefixed this with **testAddition** and not **testAdditional** other than shorter name?

Reply:
This was a typo.